ebmbot/job_configs.py

```diff
@@ -209,6 +209,24 @@
             "job_type": "delete_preview",
         }],
     },
+    "team_data": {
+        "python_file": "report_project_board.py",
```

Comment:
Where is this file?

Reply:
Well spotted, this is a mistake, thanks.